flink-ai-flow/ai_flow/ai_graph/ai_graph.py

```diff
@@ -70,6 +72,22 @@ def add_channel(self, node_id: Text, channel: Channel):
         edge = DataEdge(destination=node_id, source=channel.node_id, port=channel.port)
         self.add_edge(node_id=node_id, edge=edge)
 
+    def set_context_extractor(self, context_extractor: ContextExtractor):
+        """
+        Set the context extractor to the AIGraph.
+
+        :param context_extractor: The :class:`ContextExtractor` for the AIGraph.
```

Comment:
```
:class:`ContextExtractor`    -> :class:`~ai_flow.api.context_extractor.ContextExtractor`
```

Reply:
solved in c1785cf

Reply:
Such change should be applied to all  ":class:`ContextExtractor`"